include/dma.h

```diff
@@ -93,7 +93,7 @@ class DmaChannel {
 	                   uint8_t* const buffer);
 
 	DMA_ReservationCallback reservation_callback = {};
-	std::string_view reservation_owner           = {};
+	std::string reservation_owner                = {};
```

Comment:
Nice one here :+1:   This has been a source of bugs in the past when a `std::string_view` gets stored in some class and it outlives its owner.

Reply:
I already considered this potentially harmful code. As far as I can tell, we just eliminated the last one.